environments/production/terraform.yml

```diff
@@ -310,63 +310,63 @@ servers:
     group: "kafka"
     os: bionic
 
-  - server_name: "pgbouncer_a1-production" # host and pgbouncer_host for pl/proxy, pgbouncer host for shard dbs
+  - server_name: "pgbouncer_a1-production" # host and pgbouncer_host for pl/proxys
     server_instance_type: r5a.2xlarge
     network_tier: "db-private"
     az: "a"
     volume_size: 200
     volume_encrypted: yes
     group: "postgresql"
     os: bionic
-  - server_name: "pgbouncer_a2-production"  # former pgbouncer_host formplayer
-    server_instance_type: m5n.large
+  - server_name: "pgbouncer_a2-production"  # pgbouncer_host for pgmain,  formplayer
+    server_instance_type: t3.large
     network_tier: "db-private"
     az: "a"
     volume_size: 80
     volume_encrypted: yes
     group: "postgresql"
     os: bionic
-  - server_name: "pgbouncer_a3-production"  # pgbouncer_host formplayer
-    server_instance_type: m5n.large
+  - server_name: "pgbouncer_a3-production" # pgbouncer_host for synclogs, pgsahrds
+    server_instance_type: t3.large
     network_tier: "db-private"
     az: "a"
     volume_size: 80
     volume_encrypted: yes
     group: "postgresql"
     os: bionic
-  - server_name: "pgbouncer_b0-production"  # pgbouncer_host for synclogs
-    server_instance_type: m5n.large
+  - server_name: "pgbouncer_b0-production"  # pgbouncer_host for pgmain, formplayer
+    server_instance_type: t3.large
     network_tier: "db-private"
     az: "b"
     volume_size: 80
     volume_encrypted: yes
     group: "postgresql"
     os: bionic
-  - server_name: "pgbouncer_b1-production"
-    server_instance_type: m5n.large
+  - server_name: "pgbouncer_b1-production"  # pgbouncer_host for synclogs, pgshards
+    server_instance_type: t3.large
     network_tier: "db-private"
     az: "b"
     volume_size: 80
     volume_encrypted: yes
     group: "postgresql"
     os: bionic
-  - server_name: "pgbouncer_c0-production"  # pgbouncer_host for main, ucr
-    server_instance_type: m5n.large
+  - server_name: "pgbouncer_c0-production"  # pgbouncer_host for pgmain, formplayer
+    server_instance_type: t3.large
     network_tier: "db-private"
     az: "c"
     volume_size: 80
     volume_encrypted: yes
     group: "postgresql"
     os: bionic 
-  - server_name: "pgbouncer10-production"
+  - server_name: "pgbouncer10-production" # pgbouncer_host for ucr dbs
     server_instance_type: m5n.large
```

Comment:
Did you mean to include this one too?

Reply:
This is being handled in another PR as it is a candidate to be rebuilt in another AZ

Reply:
https://github.com/dimagi/commcare-cloud/pull/5807